crates/ink/codegen/src/generator/trait_def/call_forwarder.rs

```diff
@@ -388,6 +388,7 @@ impl CallForwarder<'_> {
             "encountered error while calling <{forwarder_ident} as {trait_ident}>::{message_ident}",
         );
         quote_spanned!(span =>
+            #( #attrs )*
```

Comment:
Maybe you want to use `cfg_attrs` here because it is generated part of the trait(not expected by the user)?